src/Paginator/HybridPaginatorAdapter.php

```diff
@@ -27,9 +27,9 @@ class HybridPaginatorAdapter extends RawPaginatorAdapter
      * @param Query                               $query       the query to search
      * @param ElasticaToModelTransformerInterface $transformer the transformer for fetching the results
      */
-    public function __construct(SearchableInterface $searchable, Query $query, ElasticaToModelTransformerInterface $transformer)
+    public function __construct(SearchableInterface $searchable, Query $query, array $options, ElasticaToModelTransformerInterface $transformer)
```

Comment:
Wouldn't it be better to pass $options as 3rd argument?

Reply:
I tried to stick to the similar functions signature that were in the same file. But yes,I guess that could cause BC break. But on the other hand it makes this function consistent with the others.

You tell me

Reply:
I guess. I'll change it now

Reply:
BC breaks are allowed in master branch as long as 6.0 is not final.